docgen/src/guides/advanced-topics.md

```diff
@@ -153,3 +153,127 @@ class App extends Component {
 export default withRouter(App);
 ```
 
+## React Native
```

Comment:
We will see but as soon as you have 15 lines of code you are good for a single page